rcsb/db/cli/RepoLoadExec.py

```diff
@@ -116,6 +117,10 @@ def main():
         help="Compare the number of loaded entries with the number expected by the holdings (for op 'pdbx_loader_check')"
     )
     parser.add_argument("--log_file_path", default=None, help="Path to runtime log file output.")
+    # args for imgs workflow format
+    parser.add_argument("--incremental_update", default=False, action="store_true", help="Ignore bcif and image time stamps and update them all.")
```

Comment:
The description should be updated to something. Proposal:
```suggestion
    parser.add_argument("--incremental_update", default=False, action="store_true", help="Whether the process should look at timestamps (see --target_file_dir and --target_file_suffix) to find a delta list of ids to update. Default is a full update.")
```